public/app/plugins/datasource/azuremonitor/types/templateVariables.ts

```diff
@@ -0,0 +1,14 @@
+export {
+  GrafanaTemplateVariableQueryType,
+  UnknownQuery,
+  AppInsightsMetricNameQuery,
+  AppInsightsGroupByQuery,
+  SubscriptionsQuery,
+  ResourceGroupsQuery,
+  ResourceNamesQuery,
+  MetricNamespaceQuery,
+  MetricDefinitionsQuery,
+  MetricNamesQuery,
+  WorkspacesQuery,
+  GrafanaTemplateVariableQuery,
+} from '../dataquery.gen';
```

Comment:
being strictly these definitions are frontend-only so in theory we don't need to define them in CUE schemas. Is there any oficial position for this kind of definitions?

Reply:
I'm not sure there is at the moment, I've defined these temporarily but with the aim of removing as many of them as possible 😊 Phase 2 of the Azure schema work is to massively clean up the types 😜